packages/ckeditor5-dev-tests/lib/tasks/runautomatedtests.js

```diff
@@ -91,11 +96,31 @@ function createEntryFile( globPatterns ) {
 		throw new Error( 'Not found files to tests. Specified patterns are invalid.' );
 	}
 
-	const filesImports = allFiles
-		.map( file => 'import "' + file + '";' )
-		.join( '\n' );
+	const entryFileContent = allFiles
+		.map( file => 'import "' + file + '";' );
+
+	if ( disallowConsoleUse ) {
+		entryFileContent.unshift( `
+const originalWarn = console.warn;
+
+beforeEach( () => {
+	console.log = ( ...data ) => {
+		originalWarn( 'Detected \`console.log()\`:', ...data );
+		throw new Error( 'Detected \`console.log()\`' );
+	};
+	console.warn = ( ...data ) => {
+		originalWarn( 'Detected \`console.warn()\`:', ...data );
+		throw new Error( 'Detected \`console.warn()\`' );
+	};
+	console.error = ( ...data ) => {
```

Comment:
Since we are covering log, warn and error it make sense to also cover `console.info` which is something that can be used similarly. I checked the reference and there's also `console.debug` 🤔  I think we can cover all five, in which case you'd probably want to do it in loop to simplify it.